source/codegen/templates/service.cpp.mako

```diff
@@ -62,6 +64,17 @@ namespace ${config["namespace_component"]}_grpc {
       output->AddAllocated(message);
     }
   }
```

Comment:
Add a blank line after this, so in generated file there is blank line after definition of `Copy` function.

Reply:
Done.